packages/@textlint/types/src/Rule/TextlintRuleContextFixCommand.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * `RuleReporterFixCommand` is intermediate object between Rule report and Kernel.
+ * `IntermediateFixCommand` has also `isAbsolute` flag value.
+ */
+export interface TextlintRuleContextFixCommand {
```

Comment:
We need to note about the difference of `TextlintRuleContextFixCommand` and `TextlintCommand`.